paperqa/settings.py

```diff
@@ -505,7 +502,7 @@ class AgentSettings(BaseModel):
     )
     index: IndexSettings = Field(default_factory=IndexSettings)
 
-    callbacks: Mapping[str, list[Callable[["EnvironmentState"], Any]]] = Field(
+    callbacks: Mapping[str, list[Callable[..., Any]]] = Field(
```

Comment:
```suggestion
    callbacks: Mapping[str, list[Callable[[Any], Any]]] = Field(
```

Let's do `Any` over `...` as the callbacks still take a single argument

And can you add a TODO comment saying to move from `Any` to `EnvironmentState`